models/user.py

```diff
@@ -32,14 +32,14 @@ class UserModel(BaseModel):
     created = db.Column(db.DateTime, default=datetime.utcnow)
 
 
-    def __init__(self, firstName, lastName, email, password, phone, role, archived):
+    def __init__(self, firstName, lastName, email, password, phone, role, archived=False):
```

Comment:
I think we can remove the parameter archive. Whenever a user is created then archived should always be set to false.

Reply:
What happen if we want to create a archived user for test?

Reply:
Personally, I'm generally against adding logic to make testing simpler. Although if testing is hard then that is a concern too. There is definitely a good reason to adjust application code to make testing simpler, however in this case I don't think it is a valid reason. This is not actually a blocker to get this PR merged, but I would be happy to discuss this more. In future refactorings this init method will probably no longer exist.